### PR TITLE
Replace old about section

### PR DESCRIPTION
**Pull request by olivier-grech**
Just a minor change to make sure the wording is the same as the README of the main repository (https://github.com/LIKO-12/LIKO-12/blob/master/README.md).

I also updated the license date.

### CODE REVIEW

**Review by Rami-Sabbagh — CHANGES_REQUESTED** [review]
Why removing the horizontal lines ?
They might be automatically added by github markdown translator, but docify doesn't add them automatically.

**Comment by olivier-grech** [comment]
Yeah, I used a macro to remove all horizontal lines and didn't noticed that. 

I intended to remove some of them, though. I only left the horizontal lines between the screenshots. I don't think the other ones are really useful.

**Review by Rami-Sabbagh — APPROVED** [review]

**Review by Rami-Sabbagh — APPROVED** [review]

**Review by Rami-Sabbagh — CHANGES_REQUESTED** [review]
After checking back, all the horizontal lines should be added back, They give better style in my view, are they really bad ?

File: README.md [hunk 1]
```diff
@@ -7,25 +7,17 @@
 [![License Badge](https://img.shields.io/badge/License-MIT-blue.svg)](?id=license)
 [![Stars Badge](https://img.shields.io/github/stars/LIKO-12/LIKO-12.svg?style=flat&label=Stars)](https://github.com/LIKO-12/LIKO-12)
 
----
```
Comment (Rami-Sabbagh) [inline README.md]: This should be added back

File: README.md [hunk 2]
```diff
@@ -7,25 +7,17 @@
...
-
 # About
 
----
```
Comment (Rami-Sabbagh) [inline README.md]: This should be added back

File: README.md [hunk 3]
```diff
@@ -7,25 +7,17 @@
...
 
-The created games are stored as disk files that can be saved, shared and played by friends and others.
+DiskOS provides an environment with basic command line programs and visual game editors.
 
----
+The created games and programs are saved as disk files that can be easily shared to friends or anyone else.
 
 # Specifications
 
----
```
Comment (Rami-Sabbagh) [inline README.md]: This should be added back

File: README.md [hunk 4]
```diff
@@ -34,12 +26,8 @@ The created games are stored as disk files that can be saved, shared and played
 | HDD   | 2x 50mb drives                       |
 | Input | Keyboard, Mouse, Touch, and Gamepads |
 
----
-
 # Features
 
----
```
Comment (Rami-Sabbagh) [inline README.md]: This should be added back

File: README.md [hunk 5]
```diff
@@ -62,56 +50,48 @@ The created games are stored as disk files that can be saved, shared and played
     - Each sfx is made of 32 notes.
     - The sfx playing speed is modifiable.
     - Supports 6 waveforms (sine, square, pulse, sawtooth, triangle, noise).
-  
----
 
 # Screenshots
 
----
```
Comment (Rami-Sabbagh) [inline README.md]: This should be added back

File: README.md [hunk 6]
```diff
@@ -62,56 +50,48 @@ The created games are stored as disk files that can be saved, shared and played
...
-
 ### The code editor
 ![Code_Editor](/_media/Code_Editor.png)
 
----
+ ---
```
Comment (Rami-Sabbagh) [inline README.md]: The whole documentation doesn't add a space before the horizontal line markdown (`---`)

File: README.md [hunk 7]
```diff
@@ -62,56 +50,48 @@ The created games are stored as disk files that can be saved, shared and played
...
 
 ### The sprite editor
 
 ![Sprite_Editor](/_media/Sprite_Editor.png)
 
----
+ ---
```
Comment (Rami-Sabbagh) [inline README.md]: The whole documentation doesn't add a space before the horizontal line markdown (`---`)

File: README.md [hunk 8]
```diff
@@ -62,56 +50,48 @@ The created games are stored as disk files that can be saved, shared and played
...
 
 ### The map editor
 ![Map_Editor](/_media/Map_Editor.png)
 
----
+ ---
```
Comment (Rami-Sabbagh) [inline README.md]: The whole documentation doesn't add a space before the horizontal line markdown (`---`)

File: README.md [hunk 9]
```diff
@@ -62,56 +50,48 @@ The created games are stored as disk files that can be saved, shared and played
...
 
 ### The SFX editor
 ![SFX_Editor](/_media/SFX_Editor.png)
 
----
+ ---
```
Comment (Rami-Sabbagh) [inline README.md]: The whole documentation doesn't add a space before the horizontal line markdown (`---`)

File: README.md [hunk 10]
```diff
@@ -62,56 +50,48 @@ The created games are stored as disk files that can be saved, shared and played
...
 
 ### The system prompt
 ![DiskOS_Prompt](/_media/DiskOS_Prompt.gif)
 
----
+ ---
```
Comment (Rami-Sabbagh) [inline README.md]: The whole documentation doesn't add a space before the horizontal line markdown (`---`)

File: README.md [hunk 11]
```diff
@@ -62,56 +50,48 @@ The created games are stored as disk files that can be saved, shared and played
...
 
 ### The BIOS POST screen
 ![BIOS_POST](/_media/BIOS_POST.png)
 
----
+ ---
```
Comment (Rami-Sabbagh) [inline README.md]: The whole documentation doesn't add a space before the horizontal line markdown (`---`)

File: README.md [hunk 12]
```diff
@@ -62,56 +50,48 @@ The created games are stored as disk files that can be saved, shared and played
...
 
 ### The BIOS setup screen
 ![BIOS_Setup](/_media/BIOS_Setup.png)
 
----
+ ---
```
Comment (Rami-Sabbagh) [inline README.md]: The whole documentation doesn't add a space before the horizontal line markdown (`---`)

File: README.md [hunk 13]
```diff
@@ -62,56 +50,48 @@ The created games are stored as disk files that can be saved, shared and played
...
 
 ### The operating system installation screen
 ![DiskOS_Installer](/_media/DiskOS_Installer.png)
 
----
-
 # License
 
----
```
Comment (Rami-Sabbagh) [inline README.md]: This should be added back

**Review by Rami-Sabbagh — CHANGES_REQUESTED** [review]
Here's the list of horizontal lines that I want back

**Review by Rami-Sabbagh — APPROVED** [review]
Everything is good now, thanks for the changes, the about section is really better now :slightly_smiling_face:

**Pull request merged**